File: app/src/main/java/com/iita/akilimo/entities/InvestmentAmount.java
package com.iita.akilimo.entities;


import io.objectbox.annotation.Entity;
import io.objectbox.annotation.Id;
import lombok.Data;

@Data
@Entity
public class InvestmentAmount {

    @Id
    long id;

    private double minInvestmentAmountUSD;
    private double minInvestmentAmountLocal;

    private double investmentAmountUSD;
    private double investmentAmountLocal;
}

File: app/src/main/java/com/iita/akilimo/entities/InvestmentAmount.kt
package com.iita.akilimo.entities

import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity(tableName = "investment_amount")
open class InvestmentAmount {

    @PrimaryKey(autoGenerate = true)
    var id: Int? = null
    var minInvestmentAmountUSD: Double = 0.0
    var minInvestmentAmountLocal: Double = 0.0
    var investmentAmountUSD: Double = 0.0
    var investmentAmountLocal: Double = 0.0
    var fieldSize: Double = 0.0
}
